include/flashinfer/attention/scheduler.cuh

```diff
@@ -24,6 +24,8 @@
 #include <sstream>
 #include <vector>
 
+#include <torch/types.h>
```

Comment:
We tend not to have a torch dependency in `include` folder (the header-only library should only depend on cuda/cutlass/stl etc, because some of the downstream frameworks (which uses the header-only library instead of python package) do not rely on torch.